### PR TITLE
Popup screen and resources for popups.

### PR DESCRIPTION
**Pull request by bartdebever**
Also created NewVersionPopup for the linking function,
Might be transformed to a other generic method if needed more.

### CODE REVIEW

File: Nanoleaf-wpf/Winleafs.Wpf.csproj [hunk 1]
```diff
@@ -43,6 +43,9 @@
     <Reference Include="Hardcodet.Wpf.TaskbarNotification, Version=1.0.5.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>packages\Hardcodet.NotifyIcon.Wpf.1.0.8\lib\net451\Hardcodet.Wpf.TaskbarNotification.dll</HintPath>
     </Reference>
+    <Reference Include="JsonMigrator, Version=1.0.1.0, Culture=neutral, processorArchitecture=MSIL">
```
Comment (StijnOostdam) [inline Nanoleaf-wpf/Winleafs.Wpf.csproj]: I don't believe this project needs reference to JsonMigrator

Comment (bartdebever) [reply Nanoleaf-wpf/Winleafs.Wpf.csproj]: For my instance, it actually does and will crash without it.
I can do more google work on this but this is a more common issue for me.

Comment (StijnOostdam) [reply Nanoleaf-wpf/Winleafs.Wpf.csproj]: Please do since this will occur in the future again with some other package if the problem is on your side

Comment (bartdebever) [reply Nanoleaf-wpf/Winleafs.Wpf.csproj]:
```
System.IO.FileNotFoundException: 'Could not load file or assembly 'JsonMigrator, Version=1.0.1.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. The system cannot find the file specified.'
```

When starting the app without this dependency. More research later

Comment (StijnOostdam) [reply Nanoleaf-wpf/Winleafs.Wpf.csproj]: Opened issue 47 for this problem: https://github.com/StijnOostdam/Winleafs/issues/47

**Pull request merged**